decidim-core/app/packs/src/decidim/attachments/file_or_link_tabs.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * This file controls the behavior of the |File| and |Link| tabs in the
+ * attachment form. It disables the |File| tab when a link is present and
+ * vice versa.
+ *
+ * Markup: decidim-admin/app/views/decidim/admin/attachments/_form.html.erb
```

Comment:
No need to add the markup, as could lose it with refactors. If someone wants to know why and when this was introduced, they can always do some archaeology with git blame/log/show 😃  

If we **really** want to check the markup this can be added as a jest test, but I think this is already covered by system/capybara specs (let me know if it's not like that, and we should add some specs covering this JS!)

```suggestion
```